src/cow-react/pages/CookiePolicy/CookiePolicy.md

```diff
@@ -1,83 +1,128 @@
-   - If you do not accept changes which are made to this Cookie Policy, or take any measures described above to opt-out by removing or rejecting cookies, you may continue to use this website but accept that it may not display and/or function as intended by us. Any social media channels connected to Gnosis Limited and third party applications will be subject to the privacy and cookie policies and practices of the relevant platform providers which, unless otherwise indicated, are not affiliated or associated with Gnosis Limited. Your exercise of any rights to opt-out may also impact how our information and content is displayed and/or accessible to you on this website and on other websites.
+
+   We may modify this Cookie Policy from time to time. If we make changes to this Cookie Policy, we will provide notice of such changes, such as by sending an email notification, providing notice through our website or updating the ‘Last Updated’ date at the beginning of this Cookie Policy. The amended Cookie Policy will be effective immediately after the date it is posted. By continuing to access or use our website after the effective date, you confirm your acceptance of the revised Cookie Policy and all of the terms incorporated therein by reference. We encourage you to review our Privacy Policy and our Cookie Policy whenever you access or use our website to stay informed about our information practices and the choices available to you.
+
+   If you do not accept changes which are made to this Cookie Policy, or take any measures described above to opt-out by removing or rejecting cookies, you may continue to use this website but accept that it may not display and/or function as intended by us. Your exercise of any rights to opt-out may also impact how our information and content is displayed and/or accessible to you on this website and on other websites.
 
 ### APPENDIX
 
 _Table: Overview of cookies placed and the consequences if the cookies are not placed_
 
+![cookie-policy-1.png](https://raw.githubusercontent.com/cowprotocol/cowswap/088e9ef3f018804be9850c0fc65fa3eea26c88d6/src/custom/assets/cow-swap/cookie-policy/cookie-policy-1.png)
```

Comment:
These links should be updated with image links from `main` branch once these images are there.